content/aips/AB128F4E15AE-AIP-CONSOLIDATE-RESERVE-FACTORS-ENABLE-DPI.md

```diff
@@ -1,63 +0,0 @@
----
```

Comment:
this prosal existed twice with two different hashes and only change of `status: WIP` vs `status: PROPOSED`